### ISSUE TITLE
`@(static)` attribute not recognized when applied to constant inside proc

### ISSUE THREAD

**Issue by mtarik34b** [odin-lang/Odin#3322]
* Operating System & Odin Version:
        Odin:    dev-2024-03:09d7f1337
        OS:      Ubuntu 22.04.4 LTS, Linux 6.5.0-26-generic
        CPU:     Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
        RAM:     15747 MiB
        Backend: LLVM 17.0.6

The `@(static)` attribute is not recognized here:
```odin
package pkg
main :: proc() {
	@(static) var :: 1
}
```
```
Error: Unknown attribute element name 'static'
        @(static) var :: 1
          ^~~~~^
        Did you forget to use build flag '-ignore-unknown-attributes'?
```

I'd expect an error message stating that applying `@(static)` to a constant inside a proc is neither supported, nor required.

**Comment by gordonshamway23** [comment]
If I'm not mistaken, the static attribute can only be applied to variables, not to compile time definitions.
The following should work:
```odin
package pkg
main :: proc() {
	@(static) var := 1
}
```

**Comment by gingerBill** [comment]
`@(static)` only works on VARIABLES, not compile time constants.

**Comment by mtarik34b** [comment]
Actually fixed with commit 8768893.
> I'd expect an error message stating that applying `@(static)` to a constant inside a proc is neither supported, nor required.
